.changeset/famous-teachers-breathe.md

```diff
@@ -0,0 +1,9 @@
+---
+'@roadiehq/backstage-plugin-home-rss': patch
```

Comment:
It introduces some new functionality so we can happily make this a minor bump.

Reply:
Done!